src/Installers/Windows/AspNetCoreModule-Setup/ANCMPackageResolver/ANCMPackageResolver.csproj

```diff
@@ -5,13 +5,11 @@
     <TargetFramework>netstandard1.0</TargetFramework>
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <RestorePackagesPath>$(RepositoryRoot).deps\ANCM</RestorePackagesPath>
```

Comment:
I think we can remove this entire project. I was only used as a dummy project to restore packages, and this project no longer has any PackageReferences.

Reply:
Yep